python/ray/_private/services.py

```diff
@@ -2179,17 +2245,29 @@ def start_ray_client_server(
     ray_client_server_host = (
         "127.0.0.1" if ray_client_server_ip == "127.0.0.1" else "0.0.0.0"
     )
-    command = [
-        sys.executable,
-        setup_worker_path,
-        "-m",
-        "ray.util.client.server",
+
+    if RAY_USES_PEX:
+        import ray
+        ray_path = ray.__path__
+        command = [
+            RAY_PEX_FILE,
+            f"{ray_path[0]}/util/client/server/server.py",
```

Comment:
I'm not familiar with pex but why do we do this differently here than other places where only sys.executable is replaced?